receiver/filereceiver/receiver.go

```diff
@@ -15,12 +15,19 @@ import (
 	"go.uber.org/zap"
 )
 
+type consumerType struct {
```

Comment:
The name `consumerType` seemed a little confused.

Do you think is it better that we just directly use `metricsConsumer`, `tracesConsumer `, `logsConsumer` in the `fileReceiver` struct ?